drud/ddev
# Improve JSON output from ddev version

## The Problem/Issue/Bug:

The JSON output from `ddev version -j` can't be parsed easily.

## How this PR Solves The Problem:

We're using the new output system to determine how to render the data (JSON or ASCII table).

I also moved the GetVersionInfo() function and it's corresponding test to the version package, since it makes a bit more sense for it to live there.

## Manual Testing Instructions:

Run `ddev version` and `ddev version -j`. Make sure the former works as it did before, and the latter should be a JSON object with individual properties for each key.

## Automated Testing Overview:

The existing test was repurposed to test `version.GetVersionInfo()`. I also added a new test to look at the output of `ddev version` directly.

## Related Issue Link(s):

https://github.com/drud/ddev/issues/568

## Release/Deployment notes:
